qa/resources/template.conf

```diff
@@ -18,8 +18,10 @@ sparkz {
     bindAddress = "%(API_ADDRESS)s:%(BIND_PORT)s"
     knownPeers = []
     agentName = "2-Hop"
-    maxConnections = %(MAX_CONNECTIONS)d
-    maxPacketSize = %(MAX_PACKET_SIZE)d
+    getPeersInterval = %(GET_PEERS_INTERVAL)d
```

Comment:
python tests are failing with the error `Unexpected exception caught during testing: %d format: a number is required, not str`. Changed at the end with s -> `getPeersInterval = %(GET_PEERS_INTERVAL)s`

Reply:
ahh, sorry, fixed it